.azure-pipelines.yml

```diff
@@ -174,3 +174,25 @@ jobs:
           _make PLATFORM=rzn1
 
           upload_cache
+
+  - job: QEMUv8_check
+    displayName: 'Run regression tests (xtest) in QEMUv8'
+    pool:
+      vmImage: ubuntu-18.04
+    container:
+      image: jforissier/optee_os_ci:qemuv8_check
+    steps:
+      - script: |
+          set -e -v
+          export LC_ALL=C
+          WD=$(pwd)
+          sudo -E bash -c "cd /root/optee_repo_qemu_v8 && repo sync"
```

Comment:
Would it make sense with a `-j` flag to `repo sync`?

Reply:
This `sync` is not expected to update a lot of things most of the time, but yes it may be worth adding `-j` still.